src/components/MobileMenuBar.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import styled from 'styled-components'
+import logo from '../assets/logo.svg'
+import menu from '../assets/menu.svg'
+import { maxWidthMediaQueries } from './Common';
+
+const MobileMenuBarContainer = styled.div`
+  display: none; 
+  padding: 15px;
+  ${maxWidthMediaQueries("mobile")} {
+    display: inline-block;
+    text-align: center;
+    width: 100%;
+    box-sizing: border-box
```

Comment:
```suggestion
    box-sizing: border-box;
```

Also I think you forgot to move padding inside the media query

Reply:
Btw I only noticed this because in your screenshot there is a horizontal scroll bar which there shouldn't be if we use border-box correctly

Reply:
oop i should remove "attention to detail" from my resume tbh

Reply:
semicolon added and padding moved

Reply:
![Screen Shot 2020-11-04 at 3 19 12 PM](https://user-images.githubusercontent.com/47487758/98178237-209ae980-1eb1-11eb-8169-a2e4469aae85.png)
Note the bottom screenshot has a scroll bar because of the Notifications text that juts out
![Screen Shot 2020-11-04 at 3 19 18 PM](https://user-images.githubusercontent.com/47487758/98178243-22fd4380-1eb1-11eb-993c-0a504eed67cf.png)